<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="4" hostname="DESKTOP-V39QH6A" failures="1" tests="3" name="Test" time="56.851" errors="0" timestamp="2022-12-27T17:41:22 IST">
  <testcase classname="com.obs.tests.LoginTest" name="verifyLogin" time="9.66"/>
  <testcase classname="com.obs.tests.VerifyUserLogin" name="verifyUserLogin" time="16.203"/>
  <testcase classname="com.obs.tests.VerifyDepositAmount" name="verifyDepositAmount" time="21.659">
    <failure type="java.lang.AssertionError" message="Deposit is not working expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Deposit is not working expected [true] but found [false]
at com.obs.pages.UserHomePage.verifyAmount(UserHomePage.java:78)
at com.obs.tests.VerifyDepositAmount.verifyDepositAmount(VerifyDepositAmount.java:78)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- verifyDepositAmount -->
  <testcase classname="com.obs.base.BasePage" name="@BeforeSuite initialyeReport" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite initialyeReport -->
  <testcase classname="com.obs.base.BasePage" name="@BeforeSuite initialyeReport" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite initialyeReport -->
  <testcase classname="com.obs.base.BasePage" name="@AfterSuite cleanup" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite cleanup -->
  <testcase classname="com.obs.base.BasePage" name="@AfterSuite cleanup" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite cleanup -->
</testsuite> <!-- Test -->
